cordapp/src/main/kotlin/net/corda/plugins/CordappUtils.kt

```diff
@@ -23,11 +25,13 @@ class CordappUtils {
         }
 
         fun createCompileConfiguration(name: String, configurations: ConfigurationContainer): Configuration {
-            return createChildConfiguration(name, configurations.single { it.name == "compile" }, configurations)
+            @Suppress("deprecation")
+            return createChildConfiguration(name, configurations.single { it.name == COMPILE_CONFIGURATION_NAME }, configurations)
         }
 
         fun createRuntimeConfiguration(name: String, configurations: ConfigurationContainer): Configuration {
-            return createChildConfiguration(name, configurations.single { it.name == "runtime" }, configurations)
+            @Suppress("deprecation")
```

Comment:
The `@Suppress` element was not here before the change. Was it?

Reply:
No, it wasn't here before. However, I'm now using Gradle's `RUNTIME_CONFIGURATION_NAME` identifier, which is marked as "deprecated" in the Javadoc because Gradle's "runtime" configuration _is_ deprecated. Hence I need to suppress the warning for referencing a deprecated field.